images/linux/scripts/installers/Install-Toolset.ps1

```diff
@@ -28,7 +28,7 @@ $ErrorActionPreference = "Stop"
 
 # Get toolset content
 $toolsetJson = Get-Content -Path "$env:INSTALLER_SCRIPT_FOLDER/toolset.json" -Raw
-$toolsToInstall = @("Python", "Node")
+$toolsToInstall = @("Python", "Node", "Go")
```

Comment:
I believe we should also change `Install-Go.ps1` script and remove `Validate-Go.ps1`?